tests/main/layout/task.yaml

```diff
@@ -36,6 +36,13 @@ execute: |
         test-snapd-layout.sh -c "test -d /var/cache/demo"
         test-snapd-layout.sh -c "test -d /opt/demo"
 
+        echo "layout constructed a mimic using tmpfs as aid"
+        [ "$(test-snapd-layout.sh -c "stat -f /opt -c '%T'")" = "tmpfs" ]
+        echo "and the tmpfs was mounted with 0755 and root/root user/group to mimic /opt in core"
+        [ "$(test-snapd-layout.sh -c "stat /opt -c '%a'")" = "755" ]
+        [ "$(test-snapd-layout.sh -c "stat /opt -c '%u'")" = "0" ]
+        [ "$(test-snapd-layout.sh -c "stat /opt -c '%g'")" = "0" ]
```

Comment:
Actually, before the defaults were 4755 (which is why I found out in practice, because software refused to run). Still, let me find a place that would be a more clear test of this.

Reply:
I found /var/spool/rsyslog with has interesting mode and ownership. Testing that now.

Reply:
"Actually, before the defaults were 4755 (which is why I found out in practice, because software refused to run). Still, let me find a place that would be a more clear test of this."

That's odd. /opt should be root:root 0755 (it is here). You might want to check your local permissions and correct them.

Reply:
My point was that tmpfs defaults are 4755, obviously opt is 755 but before this branch is merged opt would stay at tmpfs-defaults. Does this make sense?